project/Build.scala

```diff
@@ -210,7 +215,12 @@ object Build extends sbt.Build {
   lazy val irProject: Project = Project(
       id = "ir",
       base = file("ir"),
-      settings = commonIrProjectSettings
+      settings = commonIrProjectSettings ++ (
+          mimaDefaultSettings
```

Comment:
Well so actually I discovered I can simply put `mimaDefaultSettings` in our `commonSettings`, shared by all projects. Then only `previousArtifact` needs to be set in projects for which we want to check binary compat. When it's not set, it defaults to `None`, and binary compat is ignored.